src/js/gui.js

```diff
@@ -349,20 +349,18 @@ GUI_control.prototype.content_ready = function (callback) {
         log(this); // ...print out "this", which now refers to each ".cf_tip" DOM element
     });
 
-    $('.cf_tip').each(function() {
-        $(this).jBox('Tooltip', {            
-            delayOpen: 100,
-            delayClose: 100,
-            position: {
-                x: 'right',
-                y: 'center'
-            },
-            outside: 'x'
-            });
+    new jBox('Tooltip', {
```

Comment:
Is this meant to be in here?

Reply:
Yes, I will try to explain: maybe you have faced some known issue while using the Configurator: when you pass over a help icon, the tooltip appears, but sometimes, I think related with a fast movement of the cursor to other tooltip element, the tooltip does not close and it remains in the window. You can change tabs, etc. and the tooltip remains in the window until you close and open again de Configurator.

Here, that we have all the motor telemetry elements with the same tooltip next one to the other, this was happening all the time. It was very easy to reproduce, simply moving fast between motors tooltip.

Changing here the way we attach the tooltip, for some reason seem to make it better. It happens from time to time, but it is difficult to reproduce now.

So I think this change is needed, by if you prefer I can move it to a new PR.

Reply:
I think the change is good, but please do it in a different pull request - if other areas of the configurator throw up problems with this it will be easier to find that way.

Reply:
Done, new PR is https://github.com/betaflight/betaflight-configurator/pull/1604/